examples/core/quad/main.rs

```diff
@@ -266,17 +270,20 @@ fn main() {
     println!("pipelines: {:?}", pipelines);
 
     // Descriptors
-    let mut srv_pool = device.create_descriptor_pool(
-        1, // sets
-        &[pso::DescriptorRangeDesc { ty: pso::DescriptorType::SampledImage, count: 1 }],
-    );
-    let set0 = srv_pool.allocate_sets(&[&set0_layout]);
-
-    let mut sampler_pool = device.create_descriptor_pool(
-        1, // sets
-        &[pso::DescriptorRangeDesc { ty: pso::DescriptorType::Sampler, count: 1 }],
+    let mut desc_pool = device.create_descriptor_pool(
+        2, // sets
```

Comment:
Why 2 sets?

Reply:
should be one, but doesn't really matter :)